recipes/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+from common.mixins.model import TimestampMixin
+# Create your models here.
+
+class Recipe(TimestampMixin, models.Model):
+
+    EASY = 1
```

Comment:
so in Django docs, they give you two options for this kind of constants decleration. Personally, I don't like having these defined in the model, because the models can get pretty long even _without_ this kind of stuff living in them. And there's no reason for them to be defined on the model. I prefer to create a constants.py file adjacent to the models and put constants in there including these choice constants. Then they can also be reused easier, versus having to import the model to use em